contracts/priv/unity.yml

```diff
@@ -966,6 +966,34 @@ paths:
         default:
           description: Unexpected error
           $ref: '#/components/responses/ServerError'
+  /api/v2/quartz/auth/connection:
+    get:
+      operationId: GetConnection
+      tags:
+        - Connection
+      parameters:
+        - $ref: '#/components/parameters/TraceSpan'
+        - in: query
+          name: email
+          schema:
+            type: string
+            format: email
+          required: true
+          description: The email to get connection for.
+      responses:
+        '200':
+          content:
+            text/plain:
```

Comment:
I think we're going to return this as `application/json`; not sure if that will matter to the generated client code.

Reply:
ah. what's the structure going to be for this response?

Reply:
A JSON-encoded string. See https://github.com/influxdata/quartz/pull/4799 for an example curl request.